crates/router/src/connector/stripe/transformers/connect.rs

```diff
@@ -0,0 +1,633 @@
+            destination: connector_customer_id,
+            transfer_group: request.payout_id,
+        })
+    }
+}
+
+// Payouts create response transform
+impl<F> TryFrom<types::PayoutsResponseRouterData<F, StripeConnectPayoutCreateResponse>>
+    for types::PayoutsRouterData<F>
+{
+    type Error = Error;
+    fn try_from(
+        item: types::PayoutsResponseRouterData<F, StripeConnectPayoutCreateResponse>,
+    ) -> Result<Self, Self::Error> {
+        let response: StripeConnectPayoutCreateResponse = item.response;
+
+        Ok(Self {
+            response: Ok(types::PayoutsResponseData {
+                status: Some(enums::PayoutStatus::RequiresFulfillment),
+                connector_payout_id: response.id,
+                payout_eligible: None,
+                should_add_next_step_to_process_tracker: false,
+            }),
+            ..item.data
+        })
+    }
+}
+
+// Payouts fulfill request transform
+impl<F> TryFrom<&types::PayoutsRouterData<F>> for StripeConnectPayoutFulfillRequest {
+    type Error = Error;
+    fn try_from(item: &types::PayoutsRouterData<F>) -> Result<Self, Self::Error> {
+        let request = item.request.to_owned();
+        Ok(Self {
+            amount: request.amount,
+            currency: request.destination_currency,
+        })
+    }
+}
+
+// Payouts fulfill response transform
+impl<F> TryFrom<types::PayoutsResponseRouterData<F, StripeConnectPayoutFulfillResponse>>
+    for types::PayoutsRouterData<F>
+{
+    type Error = Error;
+    fn try_from(
+        item: types::PayoutsResponseRouterData<F, StripeConnectPayoutFulfillResponse>,
+    ) -> Result<Self, Self::Error> {
+        let response: StripeConnectPayoutFulfillResponse = item.response;
+
+        Ok(Self {
+            response: Ok(types::PayoutsResponseData {
+                status: Some(enums::PayoutStatus::from(response.status)),
+                connector_payout_id: response.id,
+                payout_eligible: None,
+                should_add_next_step_to_process_tracker: false,
+            }),
+            ..item.data
+        })
+    }
+}
+
+// Payouts reversal request transform
+impl<F> TryFrom<&types::PayoutsRouterData<F>> for StripeConnectReversalRequest {
+    type Error = Error;
+    fn try_from(item: &types::PayoutsRouterData<F>) -> Result<Self, Self::Error> {
+        Ok(Self {
+            amount: item.request.amount,
+        })
+    }
+}
+
+// Payouts reversal response transform
+impl<F> TryFrom<types::PayoutsResponseRouterData<F, StripeConnectReversalResponse>>
+    for types::PayoutsRouterData<F>
+{
+    type Error = Error;
+    fn try_from(
+        item: types::PayoutsResponseRouterData<F, StripeConnectReversalResponse>,
+    ) -> Result<Self, Self::Error> {
+        let response: StripeConnectReversalResponse = item.response;
+
+        Ok(Self {
+            response: Ok(types::PayoutsResponseData {
+                status: Some(enums::PayoutStatus::Cancelled),
```

Comment:
How come status is hardcoded?

Reply:
This API is for reversing the transfers made on the Connect platform. Reversing a txn on their platform does not return any explicit status in the response, so we have to rely on status codes for this. And since the response is valid, we are hardcoding status 

ref - https://docs.stripe.com/api/transfer_reversals